src/test/java/com/j256/ormlite/stmt/QueryBuilderWithSchemaTest.java

```diff
@@ -0,0 +1,1772 @@
+package com.j256.ormlite.stmt;
```

Comment:
This is a super long test which seems to be a duplicate of the QueryBuilderTest.  I'd rather you have maybe 5-10 tests that specifically test the schema code.

Reply:
Done, test was truncated.